strawberryfields/gbs/similarity.py

```diff
@@ -201,3 +210,161 @@ def event_to_sample(photon_number: int, max_count_per_mode: int, modes: int) ->
             available_modes.remove(j)
 
     return sample
+
+
+def orbit_cardinality(orbit: list, modes: int) -> int:
+    """Gives the number of samples belonging to the input orbit.
+
+    **Example usage**:
+
+    >>> orbit_cardinality([2, 1, 1], 4)
+    12
+
+    Args:
+        orbit (list[int]): orbit to count number of samples
+        modes (int): number of modes in the sample
+
+    Returns:
+        int: number of samples in the orbit
+    """
+    sample = orbit + [0] * (modes - len(orbit))
+    counts = list(Counter(sample).values())
+    return int(factorial(modes) / np.prod(factorial(counts)))
+
+
+def event_cardinality(photon_number: int, max_count_per_mode: int, modes: int) -> int:
+    """Gives the number of samples belonging to the input event.
+
+    **Example usage**:
+
+    >>> event_cardinality(6, 3, 7)
+    728
+
+    Args:
+        photon_number (int): number of photons in the event
+        max_count_per_mode (int): maximum number of photons per mode in the event
+        modes (int): number of modes in the samples
+
+    Returns:
+        int: number of samples in the event
+    """
+    cardinality = 0
+
+    for orb in orbits(photon_number):
+        if max(orb) <= max_count_per_mode:
+            cardinality += orbit_cardinality(orb, modes)
+
+    return cardinality
+
+
+def p_orbit_mc(graph: nx.Graph, orbit: list, n_mean: float = 5, samples: int = 1000) -> float:

```

Comment:
Instead of `p_orbit_mc`, how about `prob_orbit_mc`? Makes it a little easier to tell what the function does